src/suit/js/SUIT.js

```diff
@@ -33,12 +33,11 @@ var props = {
 };
 
 props = mergeObjectOnly(props, window.firefly?.app ?? {});
-const {template}= props;
 
-const lsstEntry= (label, url) => (
+const lsstEntry=
     {
-        label,
-        value: url,
+        label:'LSST RSP',
+        value: 'https://data-int.lsst.cloud/api/tap',
```

Comment:
Worried about the hard-coding here

Reply:
This should always be overridden in deployment. Therefore the values are just the default.